### PR TITLE
routing listener

### PR DESCRIPTION
**Pull request by SoloOleh**
Event listener passed route callback from file router.js

### CODE REVIEW

File: src/js/burger-menu.js [hunk 1]
```diff
@@ -23,19 +25,14 @@ document.addEventListener('DOMContentLoaded', () => {
 
   navLinks.forEach((link) => {
     link.addEventListener('click', (e) => {
+      route(e);
```
Comment (snegurova) [inline src/js/burger-menu.js]: Please pass the route function as a callback to the event listener, you don't need to call it inside callback, just pass only it

**Comment by SoloOleh** [comment]
routing listener update

File: src/js/burger-menu.js (hunk 1 above)
Comment (SoloOleh) [reply src/js/burger-menu.js]: fixed

**Comment by SoloOleh** [comment]
home-favorite button style fix

**Pull request closed without merging**